pom.xml

```diff
@@ -187,6 +187,8 @@ under the License.
                         <exclude>statefun-docs/docs/_templates/**</exclude>
                         <!-- Generated code  -->
                         <exclude>**/generated/**</exclude>
+                        <!-- Bundled license files -->
+                        <exclude>**/LICENSE*</exclude>
```

Comment:
exclude source NOTICE as well?

Reply:
@zentol It seems like NOTICE / LICENSE files without any suffixes are automatically ignored by the rat plugin already.